index.d.ts

```diff
@@ -0,0 +1,215 @@
+import HttpsProxyAgent from 'https-proxy-agent';
+
+declare module "yt-channel-info" {
+
+    enum ChannelIdType {
+        Default = 0,
+        ChannelId,
+        LegacyName,
+        CustomURL
+    }
+
+    /**
+     * A abstract ChannelInfoResponse containing a continuation string if there are more responses which can be loaded
+     * and an array of items of the type T
+     */
+    interface ChannelInfoResponseContinuation<T> {
+        items: T[];
+        continuation: string | null; // Will return null if no more results can be found.  Used with getChannelPlaylistsMore()
```

Comment:
```suggestion
        /**
        * Will be null if no more results can be found.  Used with getChannelPlaylistsMore()
        */
        continuation: string | null;
```
It's better to use multi-line comment to have the code helper on hover